packages/@react-aria/focus/src/FocusRing.tsx

```diff
@@ -40,7 +40,7 @@ export interface FocusRingProps {
  * Focus rings are visible only when the user is interacting with a keyboard,
  * not with a mouse, touch, or other input methods.
  */
-export function FocusRing(props: FocusRingProps): ReactNode {
+export function FocusRing(props: FocusRingProps): React.ReactElement<unknown, string | React.JSXElementConstructor<any>> {
```

Comment:
I think these are the defaults for the generic arguments right? So you don't need to include them?

Reply:
i'll double check, but for some reason our build previously expanded like this, so figured it was better to match that

Reply:
ok but that's not necessarily the right thing